cmd/config/scanner/scanner.go

```diff
@@ -0,0 +1,80 @@
+/*
+ * MinIO Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package scanner
+
+import (
+	"encoding/json"
+	"errors"
+	"strconv"
+
+	"github.com/minio/minio/cmd/config"
+	"github.com/minio/minio/pkg/env"
+)
+
+// Scanner sub-system constants
+const (
+	scannerDelay = "delay"
+
+	EnvScannerDelay = "MINIO_DISK_USAGE_CRAWL_DELAY"
```

Comment:
Adding ENVs that are not part of the sub-sys can get confusing, why are you bringing this in this PR? @vadmeste

Reply:
> Adding ENVs that are not part of the sub-sys can get confusing, why are you bringing this in this PR? @vadmeste

Yes, I thought it is easier to change the value using mc admin config, removing it